src/index.html

```diff
@@ -1,13 +1,65 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="
+          width=device-width,
+          user-scalable=no,
+          initial-scale=1.0,
+          maximum-scale=1.0,
+          minimum-scale=1.0
+          ">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
+
+  </head>
+  <body class="config">
+    <div class="product-card" data-qa="card">
+
+      <img
+        src="images/imac.jpeg"
+        alt="imac-photo"
+        class="product-card__image">
+      <h2 class="product-card__name config">
+        APPLE A1419 iMac 27&quot; Retina
+        <br>
+        5K Monoblock (MNED2UA/A)
+      </h2>
```

Comment:
```suggestion
    <div class="product-card" data-qa="card">
      <img
        src="images/imac.jpeg"
        alt="imac-photo"
        class="product-card__image"
      >
        
      <h2 class="product-card__name config">
        APPLE A1419 iMac 27&quot; Retina
        <br>
        5K Monoblock (MNED2UA/A)
      </h2>
```

You should add empty lines between siblings not between sibling and parent. Checklist #3.